decidim/decidim
# Fix displaying hidden resources in global search

#### :tophat: What? Why?
We are removing any search data from main index when the main resource is hidden by an administrator.

#### :pushpin: Related Issues
*Link your PR to an issue*
- Fixes #8810

#### Testing
1. Go to Frontend interface and create a meeting
2. Publish that particular meeting, then report it
3. Verify the resource is being available in the search page
4. Go to admin / Global moderations, choose hide meeting
5. Go back to the search and repeat step 3 .
6. The meeting **SHOULD NOT** be visible 

:hearts: Thank you!
